src/_4a780.c

```diff
@@ -8,7 +8,15 @@ INCLUDE_ASM("asm/nonmatchings/_4a780", func_80049B80);
 
 INCLUDE_ASM("asm/nonmatchings/_4a780", func_8004A2A4);
 
-INCLUDE_ASM("asm/nonmatchings/_4a780", func_8004A344);
+int func_8004A344(int* arg0) {
```

Comment:
the cast on arg0 isn't necessary if it's already int***, same for var_v0, but with int** instead

```c
int func_8004A344(int*** arg0) {
    int** var_v0 = arg0[7];

  if (var_v0[1] == 0) return 0;

  return *var_v0[1];
}
```